src/doc/unstable-book/src/iter-order-by.md

```diff
@@ -0,0 +1,9 @@
+# `iter_order_by`
+
+The tracking issue for this feature is: [#64295]
+
+[#64295]: https://github.com/rust-lang/rust/issues/64295
+
+------------------------
+
+Add `cmp_by`, `partial_cmp_by` and `eq_by` methods to `Iterator` in the same vein as `max_by` and `min_by`.
```

Comment:
I think you can just omit this; library features that are sufficiently documented by their doc-comments normally don't have a page in here.